src/gui/settingsdialog.cpp

```diff
@@ -275,6 +275,10 @@ SettingsDialog::SettingsDialog(ownCloudGui *gui, QWidget *parent)
     setMinimumSize(::minimumSizeHint(this));
 #ifdef Q_OS_MAC
     setActivationPolicy(ActivationPolicy::Accessory);
+#elif defined(Q_OS_WIN) && QT_VERSION < QT_VERSION_CHECK(6, 0, 0)
+    // ensure we create a native window in order to receive nativeEvents
+    // by default this is only called on show
+    create();
```

Comment:
😮

Reply:
Have you checked the behavior with Qt6?

Reply:
I just did, the create is needed there too.